frontend/src/pages/settings/components/settings-tabs/components/data-tracking/data-tracking.component.tsx

```diff
@@ -0,0 +1,41 @@
+import { Flex, Space, Switch, Typography } from 'antd';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import posthog from '@/config/posthog-config.ts';
+
+import { dispatchMessage } from '@/store/features/feedback/utils/dispatch-feedback.ts';
+
+import styles from './data-tracking.module.scss';
+
+export function DataTracking() {
+    const { t } = useTranslation();
+    const [trackingEnabled, setTrackingEnabled] = useState(() => {
```

Comment:
I think this is a setting we need to save to the DB. Current behavior is (correct me if I'm wrong)
Every time a new session starts you need to disable the tracking? This is unwanted on different levels
- Only admins can toggle and view this setting, which means non-admins always have tracking enabled
- Each time the session resets they need to remember to turn this off.
Better approach would be to globally set the tracking on / off for the whole installation of portal -> a setting in the db which can be changed with this toggle

My advise would be to remove the toggle from this PR, since the behavior is not what we want and implement the correct behavior in a second PR.

Reply:
(You can base yourself of the way the "Data Product Portal" name is settable and changeable through the general settings)

Reply:
You will need to figure out a way to fetch the value of this setting at frontend startup time, so you can always check whether to enable / disable tracking